webui/react/src/components/Transfer.tsx

```diff
@@ -194,7 +194,13 @@ const Transfer: React.FC<Props> = ({
             width="100%">
             {renderVisibleRow}
           </List>
-          <Button type="link" onClick={() => moveToLeft(filteredVisibleEntries)}>
+          <Button
+            type="link"
+            onClick={() => {
+              moveToLeft(filteredVisibleEntries);
+              // removing everything was keeping the columns out of sync with the UI...
+              moveToRight(['id', 'name']);
+            }}>
```

Comment:
Probably its not a good idea to keep `id` and `name` cuz user might think these are default columns, but we can actually get rid of them. Also, it fails test cases.